test/e2e/mcc_test.go

```diff
@@ -0,0 +1,111 @@
+package e2e_test
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/openshift/machine-config-operator/test/framework"
+	"github.com/openshift/machine-config-operator/test/helpers"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
```

Comment:
Why add an e2e test for an alert if there's a unit test? Shouldn't the unit test be sufficient?

Reply:
The unit test is good for us checking the metric, the e2e test was to make sure that monitoring would be able to retrieve it (secrets, auth, etc were all set up right). 

Long term we talked about having it talk to the thanos query endpoint to make sure monitoring actually got it (like Simon was saying here: https://github.com/openshift/machine-config-operator/pull/2802#discussion_r755919982)